### DISCUSSION TITLE
Endpoint support.

### DISCUSSION THREAD

**Discussion by AshIceLoky** [jchristn/WatsonWebsocket#100] [Q&A]
Hi,
Is it possible to create endpoints or do I have to create several servers?

**Comment by jchristn** [comment]
Hi, as of right now, you can create multiple hostnames on the same port (see ```public WatsonWsServer(List<string> hostnames, int port, bool ssl = false)```) but if you want multiple listeners with different port numbers and SSL configurations, you would have to instantiate multiple instances of ```WatsonWsServer```.
